main.tf

```diff
@@ -91,209 +111,281 @@ resource "aws_security_group_rule" "out_all" {
   to_port           = 0
   protocol          = "-1"
   cidr_blocks       = ["0.0.0.0/0"]
-  security_group_id = "${aws_security_group.db.id}"
+  security_group_id = aws_security_group.db.id
 }
 
 # The database instance itself
 resource "aws_db_instance" "db_including_name" {
-  count = "${var.database_name != "" && var.engine_type != "aurora" && var.engine_type != "aurora-mysql" && var.engine_type != "aurora-postgresql" && var.replicate_source_db == "" ? 1 : 0}"
-
-  name                        = "${var.database_name}"
-  allocated_storage           = "${var.allocated_storage}"
-  allow_major_version_upgrade = "${var.allow_major_version_upgrade}"
-  auto_minor_version_upgrade  = "${var.auto_minor_version_upgrade}"
-  backup_retention_period     = "${var.backup_retention_period}"
-  backup_window               = "${var.backup_window}"
-  copy_tags_to_snapshot       = "${var.copy_tags_to_snapshot}"
-  db_subnet_group_name        = "${local.db_subnet_group_name}"
-  engine                      = "${var.engine_type}"
-  engine_version              = "${var.engine_version}"
-  final_snapshot_identifier   = "${var.name}"
-  identifier                  = "${var.name}"
-  instance_class              = "${var.instance_class}"
-  license_model               = "${var.license_model}"
-  multi_az                    = "${var.is_multi_az}"
-  parameter_group_name        = "${aws_db_parameter_group.db.id}"
-  password                    = "${var.database_password}"
-  port                        = "${var.database_port}"
-  publicly_accessible         = "${var.publicly_accessible}"
-  vpc_security_group_ids      = ["${aws_security_group.db.id}"]
-  skip_final_snapshot         = "${var.skip_final_snapshot}"
-  storage_encrypted           = "${var.storage_encrypted}"
-  storage_type                = "${var.storage_type}"
-  username                    = "${var.database_user}"
-  tags                        = "${merge(var.tags, map("Name", format("%s-%s", var.environment, var.name)), map("Env", var.environment))}"
+  count = var.database_name != "" && var.engine_type != "aurora" && var.engine_type != "aurora-mysql" && var.engine_type != "aurora-postgresql" && var.replicate_source_db == "" ? 1 : 0
```

Comment:
nitpick: You could do something like:
```
!contains(["aurora", "aurora-mysql", "aurora-postgresql"], var.engine_type)
```

Reply:
This PR is purely to make the module compatible with 0.12. I was thinking that any actual changes/improvements would happen in a different PR

Reply:
ah ok, np